### PR TITLE
Enrich VS Code Setting Instruction (both for Data&FullStack)

### PR DESCRIPTION
**Pull request by brunolajoie**
This PR improves the VS Code setup in a partial that is used for both Web & Data setup day

### CODE REVIEW

**Comment by ssaunier** [comment]
Didn't you mean to change https://github.com/lewagon/data-setup instead? 🤔

**Comment by brunolajoie** [comment]
I think it's a [partial](https://github.com/lewagon/data-setup/blob/67837c28027bf314363c27f4f98b8cdc508d7278/build.rb#L37) that is [imported](https://github.com/lewagon/data-setup/blob/67837c28027bf314363c27f4f98b8cdc508d7278/build.rb#L105) into data_setup so I just have to **build.rb** data-setup after this PR is merged

**Comment by brunolajoie** [comment]
@krokrob @barangerbenjamin I'd like to merge this PR ahead of setup day next monday, could you let me know what you think?

I have added the minimal python extensions that I think are needed for our students in DataScience. SublimeMapping is also usefull for WebDev thought!

File: _partials/wsl2_vscode.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ We use Visual Code Studio for writing code on Windows, because it integrates nic
 For this integration to work, you need to uninstall other code editor, like for example SublimeText, as their WSL integration interferes with the integration of VS Code.
 
 ### Installation
->\- Go the [Visual Studio Code page](https://code.visualstudio.com/download)  
+>\- Go the [Visual Studio Code page](https://code.visualstudio.com/download)
```
Comment (krokrob) [inline _partials/wsl2_vscode.md:8]:
```suggestion
>\- Go to the [Visual Studio Code page](https://code.visualstudio.com/download)
```

File: _partials/wsl2_vscode.md [hunk 2]
```diff
@@ -47,19 +47,27 @@ Let's gain time now and add other extensions that will be helpful during your Bo
...
+For everyone
+- [Sublime Text Keymap](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
 
+For fullstack bootcamp:
 - [Rails Snippets](https://marketplace.visualstudio.com/items?itemName=hridoy.rails-snippets)
 - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 - [ERB Helper Tags](https://marketplace.visualstudio.com/items?itemName=rayhanw.erb-helpers)
 - [ruby-rubocop](https://marketplace.visualstudio.com/items?itemName=misogi.ruby-rubocop)
 
+For datascience bootcamp:
```
Comment (krokrob) [inline _partials/wsl2_vscode.md:63]:
```suggestion
For the Data Science bootcamp:
```

File: _partials/wsl2_vscode.md [hunk 3]
```diff
@@ -47,19 +47,27 @@ Let's gain time now and add other extensions that will be helpful during your Bo
...
->\- On the web page, click on `install`  
->\- In the browser, accept to use VS Code to install the extension  
+>\- On the web page, click on `install`
+>\- In the browser, accept to use VS Code to install the extension
 >\- In VS Code, click on `install`
 
+For everyone
+- [Sublime Text Keymap](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
 
+For fullstack bootcamp:
```
Comment (krokrob) [inline _partials/wsl2_vscode.md:57]:
```suggestion
For the Web Development bootcamp:
```

**Review by krokrob — COMMENTED** [review]
Nice @brunolajoie , I have only some suggestions

**Pull request merged**